.github/workflows/lint.yml

```diff
@@ -1,6 +1,10 @@
 name: Lint
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
```

Comment:
Would this work too?
```suggestion
  pull_request
```

Reply:
I tried that but apparently not :( ebe00a8d61f4c380a4996f80b5cc996d141d29f0